ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/SubmissionValidator.scala

```diff
@@ -312,16 +312,16 @@ object SubmissionValidator {
   type StateMap = Map[DamlStateKey, DamlStateValue]
   type LogEntryAndState = (DamlLogEntry, StateMap)
 
-  private lazy val engine = Engine()
-
   def create[LogResult](
+      engine: Engine,
```

Comment:
I'd prefer this to be added with a default parameter e.g. before `stateValueCache` parameter in this factory method. That way implementors already using `SubmissionValidator` but not wanting to share an `Engine` instance do not need to modify their code to compile after this change.

Reply:
Fixed in feafc56.